src/main/java/uk/ac/ebi/ena/webin/cli/manifest/processor/metadata/SampleProcessor.java

```diff
@@ -47,29 +55,98 @@ public Callback<Sample> getCallback() {
 
   @Override
   public void process(ValidationResult result, ManifestFieldValue fieldValue) {
-    String value = fieldValue.getValue();
-
+    String sampleValue = fieldValue.getValue();
     try {
-      SampleService sampleService =
-          new SampleService.Builder()
-              .setWebinRestV1Uri(RemoteServiceUrlHelper.getWebinRestV1Url(parameters.isTest()))
-              .setCredentials(parameters.getWebinServiceUserName(), parameters.getPassword())
-              .setWebinAuthUri(RemoteServiceUrlHelper.getWebinAuthUrl(parameters.isTest()))
-              .setBiosamplesUri(RemoteServiceUrlHelper.getBiosamplesUrl(parameters.isTest()))
-              .setBiosamplesWebinUserName(parameters.getWebinServiceUserName())
-              .setBiosamplesWebinPassword(parameters.getPassword())
-              .build();
-
-      Sample sample = ExceptionUtils.executeWithRestExceptionHandling(() -> sampleService.getSample(value),
-          WebinCliMessage.SERVICE_AUTHENTICATION_ERROR.format(SampleService.SERVICE_NAME),
-          WebinCliMessage.SAMPLE_SERVICE_VALIDATION_ERROR.format(value),
-          WebinCliMessage.SAMPLE_SERVICE_SYSTEM_ERROR.format(value));
-
+      if (isJsonSample(sampleValue)) {
+        JsonNode sampleJson = new ObjectMapper().readTree(sampleValue);
+        String action = isSampleExists(getAlias(sampleJson)) ? "MODIFY" : "ADD";
+        String sampleSubmission = generateSampleSubmissionJson(action, sampleJson);
+        if (action.equals("ADD") || (action.equals("MODIFY") && parameters.isSampleUpdate())) {
+          getSubmitService().doJsonSubmission(sampleSubmission);
```

Comment:
- Suggestion to use the xml-schema json objects. 
- Suggestion to deseriaize directly into the sample object in xml-schema json objects and attach this to the submission object and sent the submission object to:
```
getSubmitService().doJsonSubmission(<submission object from webin-xml>);
```
and